microsite/data/plugins/armo-kubescape

```diff
@@ -0,0 +1,12 @@
+title: Armo Kubescape Plugin
+author: Armo
+authorUrl: https://github.com/armosec
+category: Monitoring
+description: open-source Kubernetes Security platform, made for DevOps.
+  Our patented technology and open-source solutions fit natively within the CI/CD pipeline and existing development tools, assuring DevOps, DevSecOps, and developers that every Kubernetes cluster, container, and microservice is born and remains secure, from development to production and from configuration to run-time, every time
+documentation: https://github.com/kubescape/helm-charts#readme
```

Comment:
This doesn't seem to have anything Backstage related, is this maybe the wrong link?

Reply:
Hi @awanlin, yes I changed the readme link to the right repo one.

Thanks

Reply:
Hmmm, that doesn't seem right either. Looks like your plugin is actually here: https://github.com/armosec/backstage-plugin/tree/master/plugins/kubescape-plugin. But that doesn't have any instructions as to how to install it or what functionality it offers. These are going to really help people decide to use your plugin.

Reply:
@awanlin Thank you for your feedback. 
 I updated the readme file of the plugin with instructions of how to use it.

Thanks